database/migrations/2020_05_12_114228_rss.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+use App\Configs;
+use Illuminate\Database\Migrations\Migration;
+
+class Rss extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		defined('BOOL') or define('BOOL', '0|1');
+		defined('INT') or define('INT', 'int');
+
+		DB::table('configs')->insert([
+			[
+				'key' => 'rss_enable',
+				'value' => '0',
+				'confidentiality' => 0,
+				'cat' => 'Mod RSS',
```

Comment:
Why not `'RSS'`?

Reply:
Because all the additions in the config starts with "Mod" :stuck_out_tongue: 
Mod Frame, etc...

Reply:
It's not the first time I've thought about it... It makes them sound like add-ons rather than features.